packages/eslint-plugin-lit-a11y/tests/lib/rules/interactive-supports-focus.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @fileoverview Elements with an interactive role and interaction handlers (mouse or key press) must be focusable.
+ * @author open-wc
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const { RuleTester } = require('eslint');
+const rule = require('../../../lib/rules/interactive-supports-focus');
+
+const alwaysValid = [
+  { code: 'html`<div></div>`' },
+  { code: 'html`<div aria-hidden @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${true == true} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${true === true} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${hidden !== false} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${hidden != false} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${1 < 2} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${1 <= 2} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${2 > 1} @click=${() => void 0} />`' },
+  { code: 'html`<div aria-hidden=${2 >= 1} @click=${() => void 0} />`' },
+  { code: 'html`<div @click=${() => void 0} />`' },
+  { code: 'html`<div @click=${() => void 0} tabIndex=${undefined} />`' },
+  { code: 'html`<div @click=${() => void 0} tabIndex="bad" />`' },
+  { code: 'html`<div @click=${() => void 0} role=${undefined} />`' },
+  { code: 'html`<div role="section" @click=${() => void 0} />`' },
+  { code: 'html`<div @click=${() => void 0} aria-hidden=${false} />`' },
+  { code: 'html`<input type="text" @click=${() => void 0} />`' },
+  { code: 'html`<input type="hidden" @click=${() => void 0} tabIndex="-1" />`' },
+  { code: 'html`<input type="hidden" @click=${() => void 0} tabIndex=${-1} />`' },
+  { code: 'html`<input @click=${() => void 0} />`' },
+  { code: 'html`<input @click=${() => void 0} role="combobox" />`' },
+  { code: 'html`<button @click=${() => void 0} className="foo" />`' },
+  { code: 'html`<option @click=${() => void 0} className="foo" />`' },
+  { code: 'html`<select @click=${() => void 0} className="foo" />`' },
+  { code: 'html`<area href="#" @click=${() => void 0} className="foo" />`' },
+  { code: 'html`<area @click=${() => void 0} className="foo" />`' },
+  { code: 'html`<summary @click=${() => void 0} />`' },
+  { code: 'html`<textarea @click=${() => void 0} className="foo" />`' },
+  { code: 'html`<a @click="${showNextPage()}">Next page</a>`' },
+  { code: 'html`<a @click="${showNextPage()}" tabIndex=${undefined}>Next page</a>`' },
+  { code: 'html`<a @click="${showNextPage()}" tabIndex="bad">Next page</a>`' },
+  { code: 'html`<a @click=${() => void 0} />`' },
+  { code: 'html`<a tabIndex="0" @click=${() => void 0} />`' },
```

Comment:
`tabIndex` should be `tabindex`

Reply:
are we also checking `.tabIndex`?

Reply:
I dont think we are but we probably should... we might need to check this in the other rules, too